### DISCUSSION TITLE
Does the external link TOC icon suggest it opens in a new tab?

### DISCUSSION THREAD

**Discussion by MaxGhenis** [executablebooks/meta#132] [General] [answered]
I think I've only seen this icon to indicate that the link is opened in a new tab:
![image](https://user-images.githubusercontent.com/6076111/92416702-9f72e080-f113-11ea-9dfb-711d189d1a0a.png)

It's used for external links in the TOC, but it opens them in the same tab.

**Comment by choldgraf** [comment]
Interesting point - I had just imagined this as a way to tell readers that the link pointed to a different website instead of being a "chapter" in the book. What kind of behavior would you expect:

- That we keep the symbol and make external links open in a new tab
- That we remove this symbol *unless* the link opens in a new tab, and make non-chapter sidebar links look just like the regular ones

?

**Reply by akhmerov** [reply]
NB: Wikipedia uses this to just denote external references, including those in the same tab (see references section in any article).

**Comment by chrisjsewell** [comment]
I think that yes it would be ideal if it opened in a new tab... but technically external links are any hyperlink that points at a domain other than the domain the link exists on, which is what the symbols are depicting

**Comment by fm75** [comment]
The question of opening a new tab is discussed [here](https://ux.stackexchange.com/questions/57642/is-there-any-specific-icon-to-use-for-a-new-tab-button/57648). 

I did not expect it to open a new tab, and use the browser feature for doing it when I want to.

**Reply by chrisjsewell** [reply]
thanks

**Comment by MaxGhenis** [comment, marked as answer]
I would have initially expected it to launch in a new tab (though that's not the behavior I'd want as a creator). But based on the replies I think my expectation might not have been the norm, so the current icon is fine.

Separately, since in my case the link goes to another page in the same domain, I'd prefer to remove the icon altogether. Requested that in https://github.com/executablebooks/jupyter-book/issues/962.
